sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -197,6 +200,11 @@ describe("EventHub Receiver #RunnableInBrowser", function(): void {
       const partitionId = partitionIds[0];
       const pInfo = await client.getPartitionInformation(partitionId);
       debug(`Creating new receiver with last enqueued time: "${pInfo.lastEnqueuedTimeUtc}".`);
+      breceiver = BatchingReceiver.create(
+        (client as any)._context,
+        partitionId,
+        EventPosition.fromEnqueuedTime(pInfo.lastEnqueuedTimeUtc)
+      );
```

Comment:
`breceiver` is not used in this test. This might have gotten added when resolving merge conflicts